### ISSUE TITLE
SketchUp Motor bracket files not opening in SketchUp

### ISSUE THREAD

**Issue by seem-less** [MaslowCNC/Mechanics#87]
Hey guys,

We did not receive any Motor brackets with our package, and so we are looking for the dimensions in order to make it ourselves.  We looked at the files located at

"https://github.com/MaslowCNC/Mechanics/tree/master/Hardware"

but they don't seem to open in SketchUp web version or the Sketchup viewer.  The error message says

"This does not appear to be a Sketchup Model!"

Could these files be updated so that we can retrieve the dimensions to make it ourselves?

Thanks

**Comment by BarbourSmith** [comment]
Yes! Those files are correct, but we've had a lot of issues with them not downloading correctly from GitHub. If you download the entire repository as a zip file they will open ok for some reason 🙄 

This link is to a version in OnShape that might be easier to pull dimensions from:

https://cad.onshape.com/documents/89d090d0b25d4f65f1059990/w/8fd31c06da849c35fb0557e3/e/018b70d7517223f79421bd52

Here are the critical dimensions:

![image](https://user-images.githubusercontent.com/9359447/35114732-98b4cafe-fc3a-11e7-84b0-ad8f748fe6d6.png)
